website/index.jade

```diff
@@ -12,36 +12,69 @@ html
     section.hero
       .container
         .left
-          img(src="img/git-town-horizontal-w.svg")
-          h2 What is it?
-          p Gittown is an open-source Git plugin that makes using Git on the command line even more productive and convenient! It provides additional high-level Git commands that perform typical steps in collaborative software development workflows.
+          h2 High-level command-line interface for Git
+          p
+            | Git Town is an open-source Git plugin
+            | that provides high-level Git commands
+            | to make feature-branch based development
+            | more efficient and safe.
+
+          .buttons
+            a.button.learn-more(href="/install.html") Install
+
 
         iframe.intro-video(src="https://drive.google.com/a/originate.com/file/d/0BxFxxboeZOPlcjRiVjV5b0c4Qmc/preview" width="550" height="300" frameborder="0")
 
     section.tagline
-      h2 A better way to Git
-      p Git Town adds high-level workflow commands to Git. These commands make it super easy to create, merge, synchronize, and clean up feature branches. They are robust, safe, and well-tested.
+      h2 A workflow layer on top of vanilla Git
+      p
+        | Git is an intentionally generic and basic tool.
+        | It supports many different ways of using it equally well.
+        | It is a wonderful <i>foundation</i> for source code management.
+
+      p
+        | Using it properly as a developer is cumbersome and repetitive, though.
+        | As an example, safely merging a finished feature branch
+        | requires up to <a href="https://github.com/Originate/git-town/blob/master/features/git-ship/supplied_branch/feature_branch/with_tracking_branch.feature#L19-L33">
+        | <b>15</b></a> individual Git commands!
```

Comment:
See if you can add/remove a few words to avoid 'orphaned' words like this:
<img width="234" alt="screen shot 2015-07-27 at 9 55 41 am" src="https://cloud.githubusercontent.com/assets/1149609/8912159/bbf7ff64-3445-11e5-81eb-880e62a6ca4c.png">

(dont leave the word 'commands' alone on its own line)